bc_obps/registration/api/user_operator.py

```diff
@@ -269,7 +269,7 @@ def request_admin_access(request, payload: SelectOperatorIn):
                 return status, message
         # Making a pending UserOperator instance if one doesn't exist
         user_operator, created = UserOperator.objects.get_or_create(
-            user=user, operator=operator, role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.PENDING
+            user=user, operator=operator, role=UserOperator.Roles.PENDING, status=UserOperator.Statuses.PENDING
```

Comment:
why did you change this one to `PENDING`? I would have expected it to remain as `ADMIN` since the endpoint is only used for external users requesting prime admin access to their operator

Reply:
I changed it to be safe just incase there was some mistake and someone only did a check on a user operators role. By default all roles should set to pending just like status and then set to `admin` or `reporter` on approval.